eth/api_admin.go

```diff
@@ -137,3 +138,31 @@ func (api *AdminAPI) ImportChain(file string) (bool, error) {
 	}
 	return true, nil
 }
+
+// MevRunning returns true if mev is running
```

Comment:
`if mev is running`==>`if the validator accept bids from builder`

Reply:
fixed